pkgs/by-name/te/package.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenv,  pkgs ? import <nixpkgs> {} }:
```

Don't import `<nixpkgs>` and don't bring `pkgs`. Use `fetchFromGithub` directly.

```suggestion
{ lib, stdenv, fetchFromGithub }:
```